File: apps/s-2-admin/src/api/resetPasswordRequest/ResetPasswordRequestWhereInput.ts
import { StringFilter } from "../../util/StringFilter";
import { DateTimeFilter } from "../../util/DateTimeFilter";

export type ResetPasswordRequestWhereInput = {
  id?: StringFilter;
  createdAt?: DateTimeFilter;
  updatedAt?: DateTimeFilter;
  email?: StringFilter;
  expires?: DateTimeFilter;
};

File: apps/cactus-demo/src/APIs/ResetPasswordRequest/Dtos/ResetPasswordRequestWhereInput.cs
namespace CactusDemo.APIs.Dtos;

public class ResetPasswordRequestWhereInput
{
    public string? Id { get; set; }

    public DateTime? CreatedAt { get; set; }

    public DateTime? UpdatedAt { get; set; }

    public string? Email { get; set; }

    public DateTime? Expires { get; set; }
}
